### PR TITLE
Add category form

### PR DESCRIPTION
**Pull request by mahdiAbu3li**

### CODE REVIEW

File: src/components/AddCategoryForm/AddCategoryForm.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  openDialog: boolean;
+  setOpenDialog: (a: boolean) => void;
+  title: string;
+  buttonId: number;
+}
+export default function AddCategoryForm(props: typeProps) {
+  const { openDialog, setOpenDialog, title, buttonId } = props;
+
+  const handleClose = () => {
+      console.log(buttonId)
```
Comment (Abdallatif) [inline src/components/AddCategoryForm/AddCategoryForm.tsx]: remove console.log

File: src/components/AddCategoryForm/AddCategoryForm.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        maxWidth="sm"
+      >
+        <DialogTitle id="form-dialog-title">{title}</DialogTitle>
+        <DialogContent>
+          <DialogContentText>Category Name</DialogContentText>
+          <TextField
+            autoFocus
+            margin="dense"
+            id="name"
+            placeholder={title==="Add Category" ?"Add Category":"Edit Category" }
```
Comment (Abdallatif) [inline src/components/AddCategoryForm/AddCategoryForm.tsx]: just pass the `title` as it is

File: debug.log [hunk 3]
```diff
@@ -0,0 +1 @@
+[1031/134444.220:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
Comment (Abdallatif) [inline debug.log]: remove this file

File: src/components/AddCategoryForm/AddCategoryForm.tsx (hunk 2 above)
Comment (mahdiAbu3li) [reply src/components/AddCategoryForm/AddCategoryForm.tsx]: This is the stupidest thing I have ever done in my life

**Review by Abdallatif — APPROVED** [review]

**Pull request merged**